analysis/Dockerfile

```diff
@@ -29,6 +29,6 @@ COPY --from=build /src/analyze /usr/local/bin/analyze
 COPY --from=build /src/worker /usr/local/bin/worker
 
 # Cache analysis images.
-RUN podman pull gcr.io/ossf-malware-analysis/node && \
-    podman pull gcr.io/ossf-malware-analysis/python && \
-    podman pull gcr.io/ossf-malware-analysis/ruby
+RUN podman --storage-driver vfs pull gcr.io/ossf-malware-analysis/node && \
```

Comment:
Per https://docs.docker.com/storage/storagedriver/select-storage-driver/ vfs does not seem like a recommended thing to use.

(I'm assuming this applies to podman too)

Reply:
vfs was admittedly the first driver I managed to get working both locally and remotely. Cloudbuild seems like the optimal solution here and will hopefully remove any issues surrounding the storage driver during the build of the analysis image.